pkgs/by-name/ju/jumper/package.nix

```
@@ -0,0 +1,40 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  pkgs,
+  lib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "jumper";
+  version = "v0"; # what should I put here? No version found anywhere
+  src = fetchFromGitHub {
+    owner = "homerours";
+    repo = "jumper";
+    rev = "264f291";
+    sha256 = "sha256-6++r0mBrV+KhQc7WRiKd8l52CKJUcvzPFqe4S6O6J84=";
+  };
+
+  nativeBuildInputs = [
+    pkgs.which # removes comp warning
+  ];
+
+  buildInputs = [
+    pkgs.fzf # enables fuzzy finding
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    cp jumper $out/bin
```

```suggestion
    install -Dm755 jumper -T ''${outputBin}/bin/jumper
```

It's also possible that the installPhase recognises the binary automatically. Try removing it and see if it still works. If it doesn't copy the binary as expected, add the `install` command back.

---

@camerondugan: Sorry, the installPhase ended up being necessary after all.

---

> @camerondugan: Sorry, the installPhase ended up being necessary after all.

Really? I removed it and it still builds with the binary and everything working.

---

Uhh, I got an empty $out without the installPhase on `nixpkgs-review`.

---

Sorry, looks like outputBin won't work if this isn't a split package.

Just use the following:
```suggestion
    install -Dm755 jumper -T $out/bin/jumper
```